ompi/include/mpif-values.pl

```diff
@@ -431,26 +436,32 @@ sub write_file {
 ';
 
 sub write_fortran_file {
-    my ($header, $vals, $file) = @_;
+    my ($header, $vals, $lvals, $file) = @_;
 
     foreach my $key (sort(keys(%{$vals}))) {
         $header .= "        integer $key\n";
     }
+    foreach my $key (sort(keys(%{$lvals}))) {
+        $header .= "        integer(KIND=MPI_OFFSET_KIND) $key\n";
```

Comment:
@ggouaillardet How did you find this issue -- do you have a test case?

I *think* that we're ok in terms of backwards compatibility because this constant (`MPI_DISPLACEMENT_CURRENT`) would not have compiled when passing in to any of the displacement parameters, right?  I.e., so there must be no applications using it.

If that's correct, I think we can make this technically-backwards-compatibility-breaking change because no one could have been using it before this.

Is my analysis correct?